locales/en.default.schema.json

```diff
@@ -205,11 +200,8 @@
       "name": "Drawers"
     },
     "styles": {
-      "name": "Styles",
+      "name": "Icons",
```

Comment:
Almost feels likes if the only thing to edit about `Icons` is the color, it could potentially live in the `Colors` dropdown 🤔 
Just a thought.

Reply:
Good observation, but there will be additional settings in the future under Icons such as the thickness of icons stroke  for instance.